app/metrics.yaml

```diff
@@ -2319,11 +2319,11 @@ pocket:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
 
-installation:
+first_session:
```

Comment:
It's fine that "first session" here has underscores, but the `send_in_pings` ping is dashes, right?
first_session
first-session

Reply:
Yep! I think this is fine according to glean docs, but let me verify :)

Reply:
This is a category used to group metrics inside metrics.yaml. The place where names need to match is in `send_in_pings` and in `pings.yaml`

You can have
```yaml
performance:
  first_paint:
    kind: timespan
    send_in_pings: first-session
```

There is no connection between category and pings. It might actually be a Best Practice to choose a category name that is noticeably distinct from ping names to avoid such confusion... :thinking:

Reply:
Yep as much as it's annoying -- category names have underscores, ping names have dashes.  (This unfortunately flows from downstream requirements in BigQuery).